client/src/components/survey/PopUpContactAlert.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+const PopUpContactAlert = props => {
+  const { open, handleClose, styleButton, submitSurveyPage1, handleClickOpen } = props;
```

Comment:
We shouldn't be passing in any styling as props, either re-implement using `makeStyles` in this components or abstract into a helper file like we have for other buttons.